kinematics_interface/include/kinematics_interface/kinematics_interface_base.hpp

```diff
@@ -19,6 +19,8 @@
 #define IK_PLUGIN_BASE__IK_PLUGIN_BASE_HPP_
 
 #include <rclcpp_lifecycle/lifecycle_node.hpp>
+#include "eigen3/Eigen/Core"
+#include "eigen3/Eigen/LU"
```

Comment:
Typically <> are used for header files external to the current one. Since <> is already being used here for `rclcpp_lifecycle`, let's stick with it.

```suggestion
#include <eigen3/Eigen/Core>
#include <eigen3/Eigen/LU>
```